modules/assim.batch/R/pda.utils.R

```diff
@@ -613,21 +617,55 @@ pda.generate.knots <- function(n.knot, n.param.all, prior.ind, prior.fn, pname)
 ##'
 ##' @return Nothing. Plot is generated and saved to PDF.
 ##'
-##' @author Ryan Kelly
+##' @author Ryan Kelly, Istem Fer
 ##' @export
 pda.plot.params <- function(settings, params.subset, prior.ind) {
   # *** TODO: Generalize for multiple PFTS
   pdf(file.path(settings$pfts$pft$outdir, 
         paste0('mcmc.diagnostics.pda', settings$assim.batch$ensemble.id, '.pdf')))
-    dm <- as.mcmc(params.subset)
+   params.mcmc.list <- as.mcmc.list(lapply(params.subset, mcmc))
```

Comment:
It's something lapply requires, it needs a function with an argument regardless it is used in function's computation or not. I think I can put the whole thing in a for-loop instead of this lapply chunk. I guess it was implemented this way in the beginning for efficiency purposes, but not sure if it's serving this purpose.